jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -29,10 +29,13 @@
 
 import os
 
-if os.getenv('JAX2TF_CATEGORIZE_OUT'):
+if not os.getenv('JAX2TF_CATEGORIZE_OUT') is None:
```

Comment:
I normally see this written as "x is not None"; feels more readable, wdyt?

Reply:
Changed this in `bd4528a2`!